### PR TITLE
Release/0.1.10

### PR DESCRIPTION
**Pull request by avertrees**

### CODE REVIEW

**Review by 7emansell — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,18 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+### Updated
+
+- Updated static breadcrumbs to render correct copy (DR-3023)
+- Updated `/divisions` page to render title and content (DR-3030)
+
 ## [0.1.9] 2024-07-08
 
 ### Updated
 
+- Updates card columns to new breakpoints (DR-3036)
```
Comment (EdwinGuzman) [inline CHANGELOG.md:26]: Were these missed in the previous deployment or should they move up?

Comment (avertrees) [reply CHANGELOG.md:26]: they were missed in the changelog

Comment (avertrees) [reply CHANGELOG.md:26]: in the previous release

Comment (avertrees) [reply CHANGELOG.md:26]: or maybe I'm wrong. maybe it was just the tooltip or making the card clickable. lemme double check

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,18 +7,33 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Added
+
+- Added `/search/index` and `/collections/slug` pages (DR-3028)
+- Added `/items` pages (DR-3029)
+
+### Updated
+
+- Updated static breadcrumbs to render correct copy (DR-3023)
+- Updated `/divisions` page to render title and content (DR-3030)
```
Comment (EdwinGuzman) [inline CHANGELOG.md:20]: Please make a note that the DS was updated to 3.2.0. No ticket ref is ok.

**Review by EdwinGuzman — APPROVED** [review]
Looks good but update changelog!

File: CHANGELOG.md (hunk 1 above)
Comment (avertrees) [reply CHANGELOG.md]: updated

**Pull request merged**